python/Ganga/Core/GangaRepository/__init__.py

```diff
@@ -20,3 +20,10 @@ def getRegistries():
 
 def getRegistry(name):
     return allRegistries[name]
+
+def getRegistrySlice(name):
+    return allRegistries[name].getSlice()
+
+def getRegistryProxy(name):
+    return allRegistries[name].getProxy()
+
```

Comment:
I prefer having ```getRegistryProxy('jobs')``` to ```from Ganga.GPI import jobs``` as it allows us to drop some bad uses of ```GPI.jobs``` in places like Tasks in future. I know there are other solutions but this is hopefully a drop-in replacement.
Yes the programmer could just use ```getRegistry('...').blah()``` but this starts to get tiresome if it occurs multiple times in some section of code imo.

Maybe an argument could be made that the un-proxied slice ```getRegistrySlice``` isn't used in too many places and you could use a stripProxy with the Proxied method but I'd prefer to keep it.